chart/compass/charts/connectivity-adapter/templates/deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
               value: "https://{{ .Values.global.connectivity_adapter.tls.host }}.{{ .Values.global.ingress.domainName }}"
             - name: APP_CONNECTOR_ADAPTER_MTLS_BASE_URL
               value: "https://{{ .Values.global.connectivity_adapter.mtls.host }}.{{ .Values.global.ingress.domainName }}"
+            - name: APP_LOG_FORMAT
+              value: "kibana"
```

Comment:
Maybe it would be better to introduce in values.yaml a property called `log.format` which will contain the default log settings (only format for now) and reference this here and everywhere else where `kibana` is hardcoded.

This way if we want to switch the default log format we'll only have to do it in one place.